projects/openjdk.org/package.yml

```diff
@@ -81,16 +92,18 @@ build:
       JDK_DIR: build/*/images/jdk-bundle/jdk-{{version.marketing}}.{{version.patch}}.jdk/Contents/Home/*
       ARGS:
         - --enable-dtrace
+        # - --with-sysroot=#{MacOS.sdk_path} # from homebrew's formulae
```

Comment:
I found this additional flag missing as compared to homebrews. Any thoughts?

@jhheider

Reply:
see `gcc`:
```yaml
    darwin:
      ARGS:
        # Reliance on CLT hard path is yuck.
        - --with-sysroot=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk
```

Reply:
Sorry, do you mean I should set it or do you mean it's already set by pkgx's gcc and therefore I should not set it?

(I don't use darwin)

Reply:
if it finds it, best to omit it; if it starts panicking about the sdk, then add it in.